bridge/listener/heimdall.go

```diff
@@ -118,17 +119,27 @@ func (hl *HeimdallListener) StartPolling(ctx context.Context, pollInterval time.
 }
 
 func (hl *HeimdallListener) fetchFromAndToBlock(ctx context.Context) (uint64, uint64, error) {
-	// fromBlock - get the initial block height from config
-	chainId := hl.cliCtx.ChainID
-	fromBlock := helper.GetInitialBlockHeight(chainId)
-
-	// toBlock - get the latest block height from heimdall node
+	fromBlock := uint64(0)
 	toBlock := uint64(0)
+
 	nodeStatus, err := helper.GetNodeStatus(hl.cliCtx, ctx)
 	if err != nil {
 		hl.Logger.Error("Error while fetching heimdall node status", "error", err)
-		return fromBlock, toBlock, err
+		return 0, 0, err
+	}
+
+	chainId := hl.cliCtx.ChainID
+	if chainId == "" {
```

Comment:
Hmm I wonder if we can set or inherit the chain ID value when initalizing the bridge. The fallback check is good to have though.

Reply:
Right. I think we can fix this later. We'll track this internally, thanks!